browser/ui/webui/settings/brave_default_extensions_handler.h

```diff
@@ -23,15 +25,23 @@ class BraveDefaultExtensionsHandler : public settings::SettingsPageUIHandler {
  private:
   // SettingsPageUIHandler overrides:
   void RegisterMessages() override;
-  void OnJavascriptAllowed() override;
-  void OnJavascriptDisallowed() override;
+  void OnJavascriptAllowed() override {}
```

Comment:
why are you defining these as empty methods? What's the point of overriding them if we're not going to implement them?

Reply:
It's because they are pure virtual method in base class.